conformance/lang/lang-library/lang.int/abs.balt

```diff
@@ -0,0 +1,554 @@
+Test-Case: output
+Description: Test `abs` function with type Signed32.
+Labels: int:abs, int:Signed32
+
+public function main() {
+    int:Signed32 a1 = -2147483648;
+    int:Signed32 a2 = -1234;
+    int:Signed32 a3 = 1234;
+    int:Signed32 a4 = 2147483647;
+    int:Signed32 a5 = 0xffecda;
+    int:Signed32 a6 = -0x123fAbCD;
+
+    int b1 = a1.abs(); // @output 2147483648
+    testInt(b1);
+
+    testInt(a2.abs()); // @output 1234
+
+    int b2 = int:abs(a3);
+    testInt(b2); // @output 1234
+
+    int b3 = a4.abs();
+    testInt(b3); // @output 2147483647
+
+    testInt(a5.abs()); // @output 16772314
+
+    testInt(int:abs(a6)); // @output 306162637
+}
+
+function testInt(int i) {
+    io:println(i);
+}
+
+Test-Case: output
+Description: Test `abs` function with type Signed16.
+Labels: int:abs, int:Signed16
+
+public function main() {
+    int:Signed16 a1 = -32768;
+    int:Signed16 a2 = -1234;
+    int:Signed16 a3 = 1234;
+    int:Signed16 a4 = 32767;
+    int:Signed16 a5 = 0xacd;
+    int:Signed16 a6 = -0x4FEa;
+
+    int b1 = a1.abs();
+    testInt(b1); // @output 32768
+
+    testInt(a2.abs()); // @output 1234
+
+    int b2 = int:abs(a3);
```

Comment:
Since we're specifically testing two different ways of calling the methods, shall we include the relevant labels? function call and method call?